yb-voyager/config-templates/live-migration.yaml

```diff
@@ -159,8 +159,8 @@ target:
 assess-migration:
 
   ### Target YugabyteDB version to assess migration for (in format A.B.C.D). 
-  ### Default - latest stable version i.e. 2025.1.0.0
-  target-db-version: 2025.1.0.0
```

Comment:
this is required right now as this is the only parameter we can keep uncommented for this section